rfcs/text/0033/process.yml

```diff
@@ -51,6 +51,44 @@
 
         Best efforts are made to ensure complete lines are captured in these events. Assumptions should NOT be made that multiple lines will appear in the same event. TTY output may contain terminal control codes such as for cursor movement, so some string queries may not match due to terminal codes inserted between characters of a word.
 
+    - name: io.total_bytes_captured
+      level: extended
+      type: number
+      description: >
+        The total number of bytes captured in this event.
+
+    - name: io.total_bytes_skipped
+      level: extended
+      type: number
+      description: >
+        The total number of bytes skipped in this event.
+
+    - name: io.max_kilobytes_per_process_exceeded
```

Comment:
"max_bytes_per_process_exceeded" is more consistent with other field names in this RFC (even if the Integration config option measures it in kiB).